test/gen_source_callbacks.rb

```diff
@@ -56,18 +56,17 @@ def parse_log(input_path, yaml_path = nil)
 
   File.open(input_path, 'r') do |file|
     file.each_line.map do |line|
-      # Line format support checks
-      match = line.match(REGEXT_PRETTY)
+      match = line.match(/(?:\[(\S+)\] \(\S+\) )?(\S+): (?:{(.*)})?/)
       raise "Unsupported format for '#{line}'." unless match
-
-      head, tail = line.match(/(\S+): {(.*)/).captures
-      field_values_ts = line.match(/^\[(\S+)\]/) do |m|
-        t = Time.parse(m.to_s)
+      timestamp, head, tail = match.captures()
```

Comment:
we don't the head, so we can remove it from the regex.
And maybe use the multiline regex (like pretty) to make it more readable

Reply:
Why we do not need the head?

https://github.com/DonAurelio/metababel/blob/1034bba6517465421e95bdbc33fc547b09ca4abd/test/gen_source_callbacks.rb#L71

Reply:
Oh yeah... I'm just tired!